lipton_custom/fixtures/custom_field.json

```diff
@@ -1482,5 +1694,588 @@
+  "read_only_depends_on": null,
+  "report_hide": 0,
+  "reqd": 0,
+  "search_index": 0,
+  "translatable": 0,
+  "unique": 0,
+  "width": null
+ },
+ {
+  "allow_in_quick_entry": 0,
+  "allow_on_submit": 0,
+  "bold": 0,
+  "collapsible": 0,
+  "collapsible_depends_on": null,
+  "columns": 0,
+  "default": null,
+  "depends_on": null,
+  "description": null,
+  "docstatus": 0,
+  "doctype": "Custom Field",
+  "dt": "Purchase Receipt Item",
+  "fetch_from": "batch_no.mouthfeel",
+  "fetch_if_empty": 0,
+  "fieldname": "mouthfeel",
+  "fieldtype": "Float",
+  "hidden": 0,
+  "hide_border": 0,
+  "hide_days": 0,
+  "hide_seconds": 0,
+  "ignore_user_permissions": 0,
+  "ignore_xss_filter": 0,
+  "in_global_search": 0,
+  "in_list_view": 0,
+  "in_preview": 0,
+  "in_standard_filter": 0,
+  "insert_after": "hue",
+  "is_system_generated": 0,
+  "is_virtual": 0,
+  "label": "Mouthfeel",
+  "length": 0,
+  "mandatory_depends_on": null,
+  "modified": "2023-03-03 16:45:21.192265",
+  "module": null,
+  "name": "Purchase Receipt Item-mouthfeel",
+  "no_copy": 0,
+  "non_negative": 0,
+  "options": null,
+  "permlevel": 0,
+  "precision": "1",
+  "print_hide": 0,
+  "print_hide_if_no_value": 0,
+  "print_width": null,
+  "read_only": 1,
+  "read_only_depends_on": null,
+  "report_hide": 0,
+  "reqd": 0,
+  "search_index": 0,
+  "translatable": 0,
+  "unique": 0,
+  "width": null
+ },
+ {
+  "allow_in_quick_entry": 0,
+  "allow_on_submit": 0,
+  "bold": 0,
+  "collapsible": 0,
+  "collapsible_depends_on": null,
+  "columns": 0,
+  "default": null,
+  "depends_on": null,
+  "description": null,
+  "docstatus": 0,
+  "doctype": "Custom Field",
+  "dt": "Purchase Receipt Item",
+  "fetch_from": null,
+  "fetch_if_empty": 0,
+  "fieldname": "column_break_51",
+  "fieldtype": "Column Break",
+  "hidden": 0,
+  "hide_border": 0,
+  "hide_days": 0,
+  "hide_seconds": 0,
+  "ignore_user_permissions": 0,
+  "ignore_xss_filter": 0,
+  "in_global_search": 0,
+  "in_list_view": 0,
+  "in_preview": 0,
+  "in_standard_filter": 0,
+  "insert_after": "mouthfeel",
+  "is_system_generated": 0,
+  "is_virtual": 0,
+  "label": null,
+  "length": 0,
+  "mandatory_depends_on": null,
+  "modified": "2023-03-03 16:45:21.392808",
+  "module": null,
+  "name": "Purchase Receipt Item-column_break_51",
+  "no_copy": 0,
+  "non_negative": 0,
+  "options": null,
+  "permlevel": 0,
+  "precision": "",
+  "print_hide": 0,
+  "print_hide_if_no_value": 0,
+  "print_width": null,
+  "read_only": 0,
+  "read_only_depends_on": null,
+  "report_hide": 0,
+  "reqd": 0,
+  "search_index": 0,
+  "translatable": 0,
+  "unique": 0,
+  "width": null
+ },
+ {
+  "allow_in_quick_entry": 0,
+  "allow_on_submit": 0,
+  "bold": 0,
+  "collapsible": 0,
+  "collapsible_depends_on": null,
+  "columns": 0,
+  "default": null,
+  "depends_on": null,
+  "description": null,
+  "docstatus": 0,
+  "doctype": "Custom Field",
+  "dt": "Purchase Receipt Item",
+  "fetch_from": "item_code.tealingo.item",
```

Comment:
Invalid fetch_from
should be "item_code.tealingo_item"

Are you testing your code?